ValidationLibrary.Rules/ExtensionMethods.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ValidationLibrary.Rules
+{
+    public static class ExtensionMethods
+    {
+        public static IEnumerable<Type> AddValidationRules(this IServiceCollection service, IConfiguration config)
```

Comment:
This should return `IServiceCollection` so it acts the same way as other `ServiceCollection` extension methods.

Reply:
Done.